content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-profiles/index.yml

```diff
@@ -0,0 +1,25 @@
+pagetitle: User profiles
+description: Abstract
```

Comment:
`description` needs to contain appropriate metadata to pass all shippable checks ;-)

Reply:
Fixed